pkg/client/status_reporter.go

```diff
@@ -147,16 +147,13 @@ func (r *StatusReporter) SetFailed(statusErr error, reason string) error {
 	time := metav1.Now()
 	// The Reason should be upper case camelCase (PascalCase) according to the API docs.
 	reason = strings.ToPascalCase(reason)
+	unavailableMessage := "Rollout of the monitoring stack failed and is degraded. Please investigate the degraded status error."
```

Comment:
nit: we could put this message in a const at the top of the file

Reply:
I agree with Damien, otherwise lgtm :)